app-main/public/index.html

```diff
@@ -169,8 +169,8 @@ <h2>New Lesson</h2>
                     class="create-lesson-input"
                     type="text"
                     placeholder="Lesson title"
-                    required
                   />
+                  <span id="title-tooltip"></span>
```

Comment:
If we decided to remove the tooltip for the form validation we can keep the CSS but just remove the logic and markup for it.